src/PlaywrightEnvironment.ts

```diff
@@ -140,6 +141,8 @@ export const getPlaywrightEnv = (basicEnv = 'node') => {
             stdin.on('data', onKeyPress)
           })
         },
+        saveCoverage: async (page: Page): Promise<void> =>
+          saveCoverage(page, config.collectCoverage),
```

Comment:
A little confused about naming. Maybe we should import it with another name?

Reply:
Hm, ye. But not sure how to name it.

Reply:
> Hm, ye. But not sure how to name it.

IDK, maybe `rootSaveCoverage`?

Reply:
I choose `savePageCoverage`, should be also okay